.github/workflows/ci-feature-branch.yml

```
@@ -1,20 +1,21 @@
     steps:
-    - name: Checkout repository
-      uses: actions/checkout@v6
+      - name: Checkout repository
+        uses: actions/checkout@v6
 
-    - name: Build frontend application
-      uses: ./.github/actions/node-build
-      with:
-        node_version: '18'
+      - name: Run frontend validation and build
+        uses: ./.github/actions/node-build
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/node-build": line 4: unexpected key "type" for definition of input "node_version"

(action)